torch_geometric/nn/mesh/mesh_edge_padding.py

```diff
@@ -0,0 +1,15 @@
+import torch
+
+
+def get_meshes_edge_index(meshes):
```

Comment:
Yes, but this padding is not really used in this PR (will be in the next PR after I will resolve the issues here..). We need to pad the edge indexes since in some datasets the number of input edges could be different for each mesh example.

Reply:
@Omri-L , do you think we can drop files that are not used in this PR. should make it easier to review.

Reply:
Hi, @wsad1 , In this updated PR you have all the files that are needed. My comment from June 4th was regarding a smaller PR, but here you have it all and need all the files.
Thanks!